tools/sst_dump_test.cc

```diff
@@ -123,7 +136,11 @@ class SSTDumpToolTest : public testing::Test {
     const char* comparator_name = ikc.user_comparator()->Name();
     if (strcmp(comparator_name, ReverseBytewiseComparator()->Name()) == 0) {
       for (int32_t i = num_keys; i >= 0; i--) {
-        tb->Add(MakeKey(i), MakeValue(i));
+        if (wide_column_one_in == 0 || i % wide_column_one_in != 0) {
+          tb->Add(MakeKey(i), MakeValue(i));
+        } else {
+          tb->Add(MakeKey(i), MakeWideColumn(i));
```

Comment:
`MakeKey` here creates an internal key, which also includes the value type; we would want to update that method too so it can create internal keys with type `kTypeWideColumnEntity`. (Right now, I think we're creating regular values here too, although the content is a serialized wide-column entity.)

Note - the same applies below.